website/docs/docs/dbt-cloud-apis/metadata-querying.md

```diff
@@ -42,10 +42,16 @@ You can run ad-hoc queries directly in the [GraphQL API explorer](https://metada
 Refer to the [Apollo explorer documentation](https://www.apollographql.com/docs/graphos/explorer/explorer) for setup and authorization info. 
 
 1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. 
+
 2. Go to **Variables** at the bottom of the explorer and replace any `null` fields with your unique fields.
-3. [Authenticate](https://www.apollographql.com/docs/graphos/explorer/connecting-authenticating#authentication) via bearer auth with `YOUR_TOKEN`. Go to **Headers** at the bottom of the explorer and select **+New header**.
-4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field.
+
+3. [Authenticate](https://www.apollographql.com/docs/graphos/explorer/connecting-authenticating#authentication) via Bearer auth with `YOUR_TOKEN`. Go to **Headers** at the bottom of the explorer and select **+New header**.
+
+4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field. Remember to include the Token prefix. Your header key should look like this `{"Authorization": "Bearer <auth_token>}`.
```

Comment:
love this thank you! changed it to 'your_token' for consistency across the page.
```suggestion
4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field. Remember to include the Token prefix. Your header key should look like this `{"Authorization": "Bearer <YOUR_TOKEN>}`.
```